respokeSDK/src/main/java/com/digium/respokesdk/RespokeClient.java

```diff
@@ -191,6 +190,29 @@
 
 
     /**
+     * A listener interface to receive a notification when the request to retrieve the history
+     * of messages for a list of groups has completed
+     */
+    public interface GroupHistoriesCompletionListener {
+
+        void onSuccess(Map groupsToMessages);
```

Comment:
@samuelg @kenhunt wondering if this should actually be a generic `Map<String, List<RespokeGroupMessage>>` instead of just a `Map`. thoughts?